### PR TITLE
[WIP] Updating history for June 2023

### PR DESCRIPTION
**Pull request by bergerhoffer**
**Do not merge until after June 30** to make sure there are no more updates for the month of June.

Preview: https://file.rdu.redhat.com/~ahoffer/2023/main-june-23.html#june-2023
(VPN required; ignore old style)

### CODE REVIEW

File: supplementary_style_guide/introduction/whats-new.adoc [hunk 1]
```diff
@@ -30,6 +30,27 @@ To view the history of changes from 2020 through 2022, see the link:https://gith
...
+[[june-2023]]
+== June 2023
+
+.Glossary entries
+* xref:bimonthly[bimonthly]: Updated to "do not use" because the term can be ambiguous
+* xref:biweekly[biweekly]: Updated to "do not use" because the term can be ambiguous
+
+.Other updates
+* Added a xref:_0_9[0-9] section and moved entries starting with a number to it.
+* Added links to the new Red{nbsp}Hat link:https://redhat-documentation.github.io/accessibility-guide/[_Getting started with accessibility for writers_] guide
```
Comment (mportman12) [inline supplementary_style_guide/introduction/whats-new.adoc:45]:
```suggestion
* Added links to the new Red{nbsp}Hat link:https://redhat-documentation.github.io/accessibility-guide/[_Getting started with accessibility for writers_] guide.
```

Comment (bergerhoffer) [reply supplementary_style_guide/introduction/whats-new.adoc:45]: Thanks, updated!

**Comment by bergerhoffer** [comment]
@mportman12 Updated PR and email to include "codebase" which was merged today

**Comment by mportman12** [comment]
Updated PR to include several new merges from the end of last week.

**Review by jseseCCS — APPROVED** [review]
LGTM

**Pull request merged**